src/LuaPrinter.ts

```diff
@@ -174,7 +173,7 @@ export class LuaPrinter {
 
     public print(file: lua.File): PrintResult {
         // Add traceback lualib if sourcemap traceback option is enabled
-        if (this.options.sourceMapTraceback) {
+        if (this.options.sourceMapTraceback && !isBundleEnabled(this.options)) {
```

Comment:
Individual files in bundles don't actually call sourcemaptraceback, instead one big call is done at the end of entire bundle, so this would import unnessecarily.